static/warnings.js

```diff
@@ -0,0 +1,120 @@
+(function() {
+    function load(key, def = null) {
+        const value = window.localStorage.getItem(key)
```

Comment:
You don't like `;` right? :laughing:

Reply:
Yeah, the cases where ASI doesn't work are rare enough I feel they're unnecessary. But I can go through and add them.

Reply:
Normally we should run a minimizer so it's mostly about code coherence. We should setup eslint check. Opening an issue for that.

Reply:
Done in https://github.com/rust-lang/docs.rs/issues/2472.